lib/caseflow/service.rb

```diff
@@ -98,5 +104,39 @@ def healthcheck
     def authorized_perform(method, path, params, additional_headers = nil, options = nil)
       perform(method, path, params, DEFAULT_HEADERS.merge(additional_headers || {}), options)
     end
+
+    # Increments statsd metric and logs appeals that have one or more issues with null descriptions
+    def handle_appeals_with_null_issue_descriptions(user, appeals)
+      appeals_with_null_issue_descriptions = []
+
+      appeals.each do |appeal|
+        next unless appeal.dig('attributes', 'issues')
+
+        issues_with_null_description = appeal['attributes']['issues'].select { |issue| issue['description'].nil? }
+
+        if issues_with_null_description.any?
+          StatsD.increment("#{STATSD_KEY_PREFIX}.appeals_with_null_issue_descriptions")
+          appeals_with_null_issue_descriptions << {
+            'id' => appeal['id'],
+            'issues' => issues_with_null_description
+          }
+        end
+      end
+
+      if appeals_with_null_issue_descriptions.any?
+        log_appeals_with_no_issue_descriptions(user, appeals_with_null_issue_descriptions)
+      end
+    end
+
+    def log_appeals_with_no_issue_descriptions(user, appeals)
+      Rails.logger.warn("Caseflow returned the following appeals with null issue descriptions: #{appeals}")
+      PersonalInformationLog.create!(
+        data: {
+          user:,
+          appeals:
```

Comment:
I read in the Platform Docs that `PersonalInformationLog` can be used for PII, but what about PHI? I ask because the issue data contains PHI.

Reply:
Good question. I have reached out to Platform support and awaiting a response.

Reply:
Per @RachalCassity, they can indeed be used for both PII and PHI!